### PR TITLE
[Merged by Bors] - chore: cleanup of manifold files

### PR DESCRIPTION
**Pull request by sgouezel**
As a preparation for #19696

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Geometry/Manifold/ContMDiff/Defs.lean [hunk 1]
```diff
@@ -479,6 +479,23 @@ theorem contMDiffOn_iff :
...
     mfld_set_tac
 
+/-- zero-smoothness on a set is equivalent to continuity on this set. -/
+theorem contMDiffOn_zero_iff :
+    ContMDiffOn I I' 0 f s ↔ ContinuousOn f s := by
+  rw [contMDiffOn_iff]
+  refine ⟨fun h ↦ h.1, fun h ↦ ⟨h, ?_⟩⟩
+  intro x y
+  rw [show ((0 : ℕ∞) : WithTop ℕ∞) = 0 by rfl, contDiffOn_zero]
+  apply ContinuousOn.comp (continuousOn_extChartAt _)
```
Comment (grunweg) [inline Mathlib/Geometry/Manifold/ContMDiff/Defs.lean:489]: Can you use dot notation here, as in `  apply (continuousOn_extChartAt _).comp`?

File: Mathlib/Geometry/Manifold/ContMDiff/Defs.lean [hunk 2]
```diff
@@ -479,6 +479,23 @@ theorem contMDiffOn_iff :
...
+  · apply ContinuousOn.comp h
```
Comment (grunweg) [inline Mathlib/Geometry/Manifold/ContMDiff/Defs.lean:490]: Same here. (You could even do `apply h.comp ((continuousOn_extChartAt_symm _).mono inter_subset_left)`, right?)

**Review by grunweg — COMMENTED** [review]
Thanks for extracting this. Two nits, optional.
maintainer delegate

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**